### PR TITLE
feat: Initial setup created for the Viber Chat provider

### PR DESCRIPTION
**Pull request by firoz3130**
### What change does this PR introduce?

PR raised for the issue #4367 

### Why was this PR needed?

This is a draft PR for the viber chat provider I am working on

### Other information

I am working on the correct implementation of the D7 network. Would like to discuss on it based on the code I have written

### CODE REVIEW

File: providers/d7-viber-chat/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Novu D7ViberChat Provider
+
+A D7ViberChat chat provider library for [@novu/node](https://github.com/novuhq/novu)
+
+## Usage
+
+```javascript
+    FILL IN THE INITIALIZATION USAGE
```
Comment (Cliftonz) [inline providers/d7-viber-chat/README.md:8]: This needs to be filled in

File: providers/d7-viber-chat/package.json [hunk 2]
```diff
@@ -0,0 +1,81 @@
+{
+  "name": "@novu/d7-viber-chat",
+  "version": "0.16.3",
```
Comment (Cliftonz) [inline providers/d7-viber-chat/package.json:3]: The version needs to be updated to the .20 something like the rest of the providers

**Review by Cliftonz — CHANGES_REQUESTED** [review]

**Comment by firoz3130** [comment]
Can you review this @Cliftonz ?

**Review by Cliftonz — APPROVED** [review]

**Comment by firoz3130** [comment]
> @firoz3130 Looks like the testing pipeline failed due to the word "Firos" in the test.
> I would recommend adding it to the cspell file.

I have added that word in the cspell file 
But I haven't pushed that file here.@Cliftonz

**Comment by firoz3130** [comment]
Any things I have to add upon here @Cliftonz ?

**Comment by firoz3130** [comment]
If you can, please add hacktoberfest label in this pr?@Cliftonz

**Comment by Cliftonz** [comment]
@firoz3130 done

**Pull request closed without merging**